Apps/Src/SendTritium.c

```diff
@@ -0,0 +1,426 @@
+/**
+ * @copyright Copyright (c) 2022 UT Longhorn Racing Solar
+ * 
+ * @file SendTritium.c
+ * @brief Function implementations for the SendTritium application.
+ * 
+ * This contains functions relevant to updating the velocity and current setpoitns
+ * of the Tritium motor controller. The implementation includes a normal current
+ * controlled mode, a one-pedal driving mode, cruise control, and regenerative braking.
+ * The logic is determined through a finite state machine implementation.
+ * 
+ * @author Nathaniel Delgado (NathanielDelgado)
+ * @author Diya Rajon (diyarajon)
+ * @author Ishan Deshpande (IshDeshpa)
+ */
+
+#include "Pedals.h"
+#include "ReadCarCAN.h"
+#include "Minions.h"
+#include "SendTritium.h"
+#include "ReadTritium.h"
+#include "CANbus.h"
+
+
+// Macros
+#define MAX_VELOCITY 50.0f // m/s    if rmp: 20000.0f // rpm
```

Comment:
We do need to figure out whether it's RPM or m/s, and ensure that we're sending and receiving the right data (+ converting units when we need to).
[Here's the link to the Controls Sharepoint folder.](https://utexas.sharepoint.com/:f:/s/ENGR-LonghornRacing/ElA5vebiiyFNltiZD4OtSaQByIxLEAP3tlJlGAjd56zD3A?e=nYh0py)
[Here's the document with CAN messages and their units.](https://utexas.sharepoint.com/:b:/s/ENGR-LonghornRacing/EVb2-iGdYkdGkEMl44-lTvIBgV-iotwTGzEtiI2kKI31mA?e=nswyrD)
Look for the Motor Drive Command (which we send the current/vel setpoint in) and the Velocity Measurement Command (which we get the velocity in). Velocity Measurement will end up being done in ReadCarCAN, so talk to Gage/Arnav (you may have to wait until they're done and then merge their branch back into your branch, but communicate with them). At the end of the day though, we do need to be sending/receiving correct units.